### PR TITLE
Making changes to xarray COH for Pf2

### PR DESCRIPTION
**Pull request by andrewram4287**

### CODE REVIEW

File: sccp/imports/CoH.py [hunk 1]
```diff
@@ -164,57 +126,6 @@ def CoH_xarray(numCells, cond, allmarkers):
...
-    "CD4 TEMRA": 16,
-    "CD20 B": 17,
-    "CD20 B Naive": 18,
-    "CD20 B Memory": 19,
-    "CD33 Myeloid": 20,
-    "Classical Monocyte": 21,
-    "NC Monocyte": 22,
-}
 
 marker_dict_all = [
```
Comment (andrewram4287) [inline sccp/imports/CoH.py:130]: @aarmey Should I be using all markers in the flow cytometry, or only pSTAT and surface markers?

Comment (aarmey) [reply sccp/imports/CoH.py:130]: Given that scRNAseq essentially includes all the markers, and has worked best so far for Pf2, let's include them all.

**Review by aarmey — CHANGES_REQUESTED** [review]
How are you normalizing? The factorization is only explaining FSC/SSC, which indicates it is almost certainly off.

**Comment by andrewram4287** [comment]
@aarmey Should be fixed now!

**Comment by aarmey** [comment]
Ok, I'm making some other changes

File: sccp/figures/figure5.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    # Import of single cells: [Patient, Treatment, Cell, Marker]
+    cohXA, celltypeXA = CoH_xarray(saveXA=False)
+
+    # Shrink dataset
+    cohXA = cohXA.loc[:, :, :500, :]
+    celltypeXA = celltypeXA.loc[:, :, :500]
+
+    # Normalize here
+    cohXA.values /= np.nanmean(cohXA.values, axis=0)
```
Comment (aarmey) [inline sccp/figures/figure5.py:27]: I'm not sure if this is the right normalization, but you should just perform the normalization here. This is a very fast operation, and it means you don't have to rebuild the data every time this changes.

**Comment by aarmey** [comment]
@andrewram4287 I reformatted this quite a bit. Please rebuild the xarray dataset, check that this runs, and then ping me to review.

**Comment by aarmey** [comment]
(Note that I had to overwrite your changes. It looked like you fixed an indexing issue which I may have brought back here.)

**Comment by andrewram4287** [comment]
@aarmey Should be good now! It seems like the first about 300 cells are usually T cells, should I randomize which cells we obtain?

**Review by aarmey — CHANGES_REQUESTED** [review]
Look at the build errors first.

**Comment by aarmey** [comment]
Great! Merging.

**Pull request merged**